source/Halibut.Tests/Builders/RedisPendingRequestQueueBuilder.cs

```diff
@@ -59,7 +59,7 @@ public QueueHolder Build()
 
             var disposableCollection = new DisposableCollection();
 
-            var redisFacade = new RedisFacade("localhost:" + RedisTestHost.Port(), (Guid.NewGuid()).ToString(), log);
+            var redisFacade = RedisFacadeBuilder.CreateRedisFacade(port: RedisTestHost.Port());
```

Comment:
Doing this means we can run the tests on a machine that doesn't have redis/docker and instead point it to another machine which does.